ui/app/templates/vault/cluster/secrets/backends.hbs

```diff
@@ -7,106 +7,115 @@
-      </Li.menu>
-    </LinkableItem>
-  {{/let}}
-{{/each}}
-
-{{#each this.unsupportedBackends as |backend|}}
-  <LinkableItem data-test-secret-backend-row={{backend.id}} @disabled={{true}} as |Li|>
-    <Li.content
-      @accessor={{if (eq backend.version 2) (concat "v2 " backend.accessor) backend.accessor}}
-      @description={{backend.description}}
-      @glyphText={{backend.engineType}}
-      @glyph={{or (if (eq backend.engineType "kmip") "secrets" backend.engineType) "secrets"}}
-      @title={{backend.path}}
-    />
-    <Li.menu>
-      <PopupMenu name="engine-menu">
+{{#each this.sortedDisplayableBackends as |backend|}}
+  <LinkedBlock
+    @params={{array backend.backendLink backend.id}}
+    class="list-item-row linkable-item is-no-underline"
+    data-test-auth-backend-link={{backend.id}}
+    @disabled={{if backend.isSupportedBackend false true}}
+  >
+    <div class="linkable-item-content" data-test-linkable-item-content>
+      <div class="has-text-grey">
+        {{#if backend.icon}}
+          <ToolTip @horizontalPosition="left" as |T|>
+            <T.Trigger>
+              <Icon @name={{backend.icon}} class="has-text-grey-light" data-test-linkable-item-glyph />
+            </T.Trigger>
+            <T.Content @defaultClass="tool-tip">
+              <div class="box">
+                {{or backend.engineType backend.path}}
+              </div>
+            </T.Content>
+          </ToolTip>
+        {{/if}}
+        {{#if backend.path}}
+          {{#if backend.isSupportedBackend}}
+            <LinkTo
+              @route={{backend.backendLink}}
+              @model={{backend.id}}
+              class="has-text-black has-text-weight-semibold"
+              data-test-secret-path
+            >
+              {{backend.path}}
+            </LinkTo>
+          {{else}}
+            <span data-test-secret-path>{{backend.path}}</span>
+          {{/if}}
+        {{/if}}
+      </div>
+      {{#if backend.accessor}}
+        <code class="has-text-grey is-size-8" data-test-linkable-item-accessor>
+          {{backend.accessor}}
+        </code>
+      {{/if}}
+      {{#if backend.description}}
+        <ReadMore data-test-linkable-item-description>
+          {{backend.description}}
+        </ReadMore>
+      {{/if}}
+      {{yield}}
+    </div>
+    {{! meatball sandwich menu }}
+    <div class="linkable-item-menu" data-test-linkable-item-menu={{backend.path}}>
+      <PopupMenu @name="engine-menu">
         <Confirm as |c|>
-          <nav class="menu" aria-label="unsupported secrets engine menu">
+          <nav class="menu" aria-label="{{if backend.isSupportedBackend 'supported' 'unsupported'}} secrets engine menu">
             <ul class="menu-list">
               <li class="action">
-                <LinkTo @route="vault.cluster.secrets.backend.configuration" @model={{backend.id}} data-test-engine-config>
+                <LinkTo @route="vault.cluster.secrets.backend.configuration" @model={{backend.id}}>
```

Comment:
This route might be different for engines like pki and kmip, no?

Reply:
Good question. That is an issue when you click on the backend row. So for PKI for example it takes you to the overview page. But the configuration page is the same for all. I confirmed on main.